src/lib/jobs/report_subscriptions.rs

```diff
@@ -21,27 +19,33 @@ pub async fn report_subscriptions_to_cj(
         .await
         .expect("Could not retrieve subscriptions from DB.");
     statsd.gauge(
-        &TraceType::ReportSubscriptions,
-        "n-not-reported",
+        &LogKey::ReportSubscriptionsNNotReported,
         not_reported_subscriptions.len(),
     );
+
     for sub in not_reported_subscriptions {
         let next_status = match sub.aic_expires {
             Some(aic_expires) => {
                 if aic_expires < sub.subscription_created {
-                    println!(
-                        "Affiliate cookie expired before subscription {} created. Will not report.",
-                        &sub.id
+                    // TODO info or error?
+                    error_and_incr!(
+                        statsd,
+                        LogKey::ReportSubscriptionsAicExpiredBeforeSubscriptionCreated,
+                        sub_id = &sub.id.to_string().as_str(),
+                        "AIC expired before subscription created. Will not report."
                     );
                     Status::WillNotReport
                 } else {
                     Status::Reported
                 }
             }
             None => {
-                println!(
-                    "Sub {} does not have an aic expires. Will not report.",
-                    &sub.id
+                // TODO info or error?
+                error_and_incr!(
```

Comment:
Should this be info?

Reply:
No. This is not an expected case. Keeping it as error.